<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="originalVideo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="changedVideo">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>270</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="changeButton">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>10</y>
      <width>260</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>210 구름고딕OTF 070</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>INSERT
FILTER</string>
    </property>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>209</y>
      <width>170</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>210 구름고딕OTF 070</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>ORIGINAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="text2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>210</y>
      <width>180</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>210 구름고딕OTF 070</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CHANGED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>260</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>210 구름고딕OTF 070</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SAVE IMG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>10</y>
      <width>260</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>210 구름고딕OTF 070</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>LOAD IMG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>841</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 도현</family>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>무엇이든</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 도현</family>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>바꿔드립니다</string>
    </property>
   </widget>
   <zorder>loadButton</zorder>
   <zorder>originalVideo</zorder>
   <zorder>changedVideo</zorder>
   <zorder>changeButton</zorder>
   <zorder>text</zorder>
   <zorder>text2</zorder>
   <zorder>saveButton</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
